tests/test_config.py

```diff
@@ -74,6 +74,43 @@ def test_core_config(app, status, warning):
     assert cfg['project'] == cfg.project == 'Sphinx Tests'
 
 
+@pytest.mark.sphinx(testroot='config_yaml', confoverrides={
+    'root_doc': 'root',
+    'nonexisting_value': 'True',
+    'latex_elements.maketitle': 'blah blah blah',
+    'modindex_common_prefix': 'path1,path2'})
```

Comment:
It would be better to separate the core feature of the config object to another cases (overrides, `in` API, and so on). They're not related at all.

Reply:
is it enough to just remove the overrides and only keep the initial tests?

```
    # simple values
    assert 'project' in cfg.__dict__
    assert cfg.project == 'Sphinx <Tests>'
    assert cfg.templates_path == ['_templates']
```

Reply:
Yes :-)

Reply:
done 👍